packages/core/src/hosted-form/hosted-form-options.ts

```diff
@@ -38,6 +41,11 @@ export interface HostedStoredCardFieldOptionsMap {
     [HostedFieldType.CardNumberVerification]?: HostedStoredCardFieldOptions;
 }
 
+type BlueSnapDirectHostedCardFieldOptionsMap = Pick<
+    HostedCardFieldOptionsMap,
+    HostedFieldType.CardName
+>;
+
```

Comment:
This file will be deprecated in favor of `packages/payment-integration-api/src/hosted-form/hosted-form-options.ts`. As a result, any modification is discouraged.

In addition, we wish to keep this file free of payment method specifics. Could you possibly put the BlueSnap material into its own package?

Reply:
I know, but since they are not fully decoupled yet, I had to modify both files. Otherwise, we get an error:

<pre>
ERROR in packages/core/src/payment/strategies/credit-card/credit-card-payment-strategy.ts(63,59)
        TS2345: Argument of type 'import("packages/<strong>payment-integration-api</strong>/src/hosted-form/<strong>hosted-form-options</strong>").default' is not assignable to parameter of type 'import("packages/<strong>core</strong>/src/hosted-form/<strong>hosted-form-options</strong>").default'.
<pre/>

Reply:
Undone.